Testing/VelaptorTests/Content/TexturePathResolverTests.cs

```diff
@@ -68,22 +69,22 @@ public void ResolveFilePath_WhenContentItemDoesNotExist_ThrowsException()
         // Arrange
         var mockDirectory = new Mock<IDirectory>();
         mockDirectory.Setup(m => m.GetFiles(this.atlasContentDir, "*.png"))
-            .Returns(() =>
-            {
-                return new[]
+            .Returns(
+                new[]
                 {
                     $"{this.baseDir}/other-file-A.png",
-                    $"{this.baseDir}/other-file-B.txt",
-                };
-            });
+                    $"{this.baseDir}/other-file-B.txt"
+                }
+            );
```

Comment:
Please move the ending parenthesis for the `Return()` method on the same line as the ending curly brace as the array initializer.  Also please move the `new[] to the end of the same line as the `.Returns(`.

This is shown as a warning by the analyzers.

```suggestion
        mockDirectory.Setup(m => m.GetFiles(this.atlasContentDir, "*.png"))
            .Returns(new[]
                {
                    $"{this.baseDir}/other-file-A.png",
                    $"{this.baseDir}/other-file-B.txt"
                });
```